### PR TITLE
2540 tooltip component

### LINKED ISSUE #2540: Help tooltip component
As a user, 
I often want more context about certain UI elements, 
so I can learn better interact with the site.

----

**Design**

A commonly used help icon which reveals a tooltip with additional context will help in many places across the site.

screen shot 2018-04-30 at 11 01 53 pm

**Acceptance criteria**

- Create a new tooltip component 
- On click, the tooltip should be revealed and the icon should change color.
- Once active, clicking or tapping anywhere (including back on the icon) will dismiss the tooltip.
- Once ready, push the component next to a random UI element on dev, and show it to me.

--------------------

Note: We'll start using this on real elements in future sprints.

### PR DESCRIPTION
**Pull request by gl2748**
Closes #2540

### CODE REVIEW

File: src/app/components/modules/UserWallet.jsx [hunk 1]
```diff
@@ -574,6 +576,18 @@ class UserWallet extends React.Component {
...
+                            content={[
+                                <h4 key="sp-helptip">What is Steem Power?</h4>,
+                                <p key="sp-helptip-1">
+                                    'Steem Power - Steem Power (abbreviated SP)
+                                    is a measurement of how much influence a
+                                    user has in the Steem network.'
+                                </p>,
+                            ]}
+                        >
+                            <IconButton size="small" fill="green" />
```
Comment (bnchdrff) [inline src/app/components/modules/UserWallet.jsx]: the icon shouldn't be a pencil, it should be a question mark as speced in https://github.com/steemit/condenser/issues/2540

File: src/app/components/elements/HelpTip/index.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            visible: false,
+        };
+    }
+
+    show = () => this.setVisibility(true);
+    hide = () => this.setVisibility(false);
+
+    setVisibility = visible => {
+        this.setState(
+            Object.assign({}, this.state, {
```
Comment (jnordberg) [inline src/app/components/elements/HelpTip/index.jsx]: Is this needed? I thought react's setState would do something similar behind the scenes

Comment (gl2748) [reply src/app/components/elements/HelpTip/index.jsx]: You're right, it's not needed.

File: src/app/components/modules/UserWallet.jsx (hunk 1 above)
Comment (gl2748) [reply src/app/components/modules/UserWallet.jsx]: Ok I have done thate.

**Comment by gl2748** [comment]
tooltip

**Comment by gl2748** [comment]
@bnchdrff I added this as a demo implementation for review, removing from the PR.

**Pull request merged**